app/main/forms/cancel.py

```diff
@@ -0,0 +1,22 @@
+from flask.ext.wtf import Form
+from wtforms import RadioField, validators
+
+
+class CancelBriefForm(Form):
+    """Form for the buyer to tell us why they cancel the requirement
+    """
+    choices = [
+        ('cancel', 'The requirement has been cancelled'),
+        ('unsuccessful', 'There were no suitable suppliers'),
+    ]
+    cancel_reason = RadioField(
+        validators=[validators.InputRequired(message='Answer required')],
```

Comment:
Might be worth double checking with content if this is our default message (if you haven't already). I swear I've seen 'You must answer this question' elsewhere

Reply:
That is used in some places. Notably in conjunction with the multi question - dependent question stuff. The outlier being the `company_summary.html` and the `ReuseDeclarationForm` which I created ages ago :|

'Answer required' is also used in some places but *drumroll please* the winner is, by a wide margin...
'You need to answer this question.' with 101+ matches in 100+ files!!! 🎉 

I'll get that changed.